src/gConsent/manage/approveAccessRequest.ts

```diff
@@ -305,29 +453,17 @@ export async function approveAccessRequest(
       );
     }
 
-    return accessGrant;
+    return accessGrant as AccessGrant;
```

Comment:
Why is the type assertion required despite the type guards above?

Reply:
Good catch - this is indeed no longer necessary.